src/slider.stories.ts

```diff
@@ -0,0 +1,286 @@
+import { UPDATE_STORY_ARGS } from '@storybook/core-events';
+import { addons } from '@storybook/preview-api';
+import { html, nothing } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import SliderComponent from './slider.js';
+
+const meta: Meta = {
+  title: 'Slider',
+  decorators: [
+    withActions,
+    (story) =>
+      html`<form action="/">
+        <script type="ignore">
+          import '@crowdstrike/glide-core/slider.js';
+        </script>
+
+        ${story()}
+      </form>`,
+  ],
+  parameters: {
+    actions: {
+      handles: ['change', 'input', 'invalid'],
+    },
+    docs: {
+      story: {
+        autoplay: true,
+      },
+    },
+  },
+  args: {
+    label: 'Label',
+    'addEventListener(event, handler)': '',
+    'checkValidity()': '',
+    disabled: false,
+    'hide-label': false,
+    max: '',
+    min: '',
+    multiple: false,
```

Comment:
Nit: curious about the choice to use the world `multiple` vs something like `range`.

Reply:
I chose `multiple` to align with Dropdown, as we tend to use the same attribute names for similar functionality. But I'm curious what you and @clintcs think too. Does `range` make more sense here?

Reply:
I don't think it would be a problem to deviate from Dropdown. Dropdown uses`multiple` attribute because native uses `multiple` with `<select>`. And native ranges don't have a `multiple` attribute.

But I think `multiple` makes sense here. `range`, in terms of native, isn't an attribute in itself and, even with `type="range"`, it doesn't mean multiple ranges. It just means "slider".